### DISCUSSION TITLE
HW 5 Number 1 Part b

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#65] [Q&A]
Not sure why the central difference derivative is so different than the other two. Hoping for some clarification.

![image](https://user-images.githubusercontent.com/106542680/176456542-1db08203-b5fa-4b7d-a5c1-340c4867a802.png)

**Comment by cooperrc** [comment]
It shouldn't be that different. 

The difference between forward and backward should be the same numbers but shifted left or right (along the x-axis)

The central difference should be the average of the two values, but in the middle.

**Reply by feliciones** [reply]
Thank you for asking this clarifying question @ecieszynski  , if done correctly the shift is the same for forward and backward?
